src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AuroraAzureApimSubPart.kt

```diff
@@ -5,47 +5,42 @@ import com.fkorotkov.kubernetes.newObjectMeta
 import no.skatteetaten.aurora.boober.feature.name
 import no.skatteetaten.aurora.boober.feature.namespace
 import no.skatteetaten.aurora.boober.model.AuroraConfigFieldHandler
-import no.skatteetaten.aurora.boober.model.AuroraContextCommand
 import no.skatteetaten.aurora.boober.model.AuroraDeploymentSpec
 import no.skatteetaten.aurora.boober.model.AuroraResource
-import no.skatteetaten.aurora.boober.model.findSubKeys
 import no.skatteetaten.aurora.boober.model.openshift.ApimSpec
 import no.skatteetaten.aurora.boober.model.openshift.AuroraApim
 import no.skatteetaten.aurora.boober.service.AuroraDeploymentSpecValidationException
 import no.skatteetaten.aurora.boober.utils.boolean
-import no.skatteetaten.aurora.boober.utils.isValidDns
-import no.skatteetaten.aurora.boober.utils.startsWithSlash
+import no.skatteetaten.aurora.boober.utils.isListOrEmpty
+import no.skatteetaten.aurora.boober.utils.pattern
+import no.skatteetaten.aurora.boober.utils.startsWith
 import no.skatteetaten.aurora.boober.utils.validUrl
 
 val AuroraDeploymentSpec.isApimEnabled: Boolean
     get() {
         return this.getOrNull(AuroraAzureApimSubPart.ConfigPath.enabled) ?: false
     }
 
-val AuroraDeploymentSpec.azureApimPolicies: Map<String, Boolean>?
-    get() {
-        //  Somewhat strange construction in order to avoid error with null as typeCast
-        val rawGroup = this.getOrNull<Any>(AuroraAzureApimSubPart.ConfigPath.policies) ?: return null
-        return rawGroup as Map<String, Boolean>
-    }
-
 class AuroraAzureApimSubPart {
     object ConfigPath {
         private const val root = "azure/apim"
         const val enabled = "$root/enabled"
+        const val apiName = "$root/apiName"
+        const val version = "$root/version"
         const val path = "$root/path"
-        const val openapiUrl = "$root/openapiUrl"
-        const val serviceUrl = "$root/serviceUrl"
+        const val openApiURL = "$root/openApiURL"
+        const val serviceURL = "$root/serviceURL"
         const val policies = "$root/policies"
-        const val apiHost = "$root/apiHost"
```

Comment:
Denne blir nok lagt til senere, men kanskje ikke akkurat slik. For ikke å ha ting som ikke virker i Boober, tar jeg den nå bort.